diff -Naru ../orig/config.sub ./config.sub
--- ../orig/config.sub	2018-11-26 14:23:59.224388851 +0100
+++ ./config.sub	2018-11-26 14:24:55.300412850 +0100
@@ -1416,6 +1416,7 @@
 	      | -morphos* | -superux* | -rtmk* | -rtmk-nova* | -windiss* \
 	      | -powermax* | -dnix* | -nx6 | -nx7 | -sei* | -dragonfly* \
 	      | -skyos* | -haiku* | -rdos* | -toppers* | -drops* | -es* \
+	      | -jehanne* \
 	      | -onefs* | -tirtos* | -phoenix* | -fuchsia* | -redox*)
 	# Remember, each alternative MUST END IN *, to match a version number.
 		;;
